src/UI/Denominate/tests/denominateComponent.test.tsx

```diff
@@ -22,7 +22,8 @@ describe('Denominate component when decimals = 2', () => {
       value: '9999979999800000000000000',
       showLastNonZeroDecimal: false,
       showLabel: true,
-      decimals: 2
+      decimals: 2,
+      egldLabel: 'EGLD'
```

Comment:
We should probably import this from constants

Reply:
it's OK since it's a test file